src/beautify.coffee

```diff
@@ -168,11 +168,14 @@ beautify = ({ editor, onSave, language }) ->
 
     # Get current editor's text
     text = undefined
+    textIsNotEntireFile = false
     if not forceEntireFile and isSelection
       text = editor.getSelectedText()
+      textIsNotEntireFile = editor.getSelectedText().length < editor.getText().length
```

Comment:
How is `isSelection` different from the intention of `textIsNotEntireFile`?

Reply:
You can select the entire file and beautify it. In this case you don't want the selection to be treated the same as any arbitrary code chunk.

For instance, if one day we have https://github.com/avh4/elm-format/issues/65 resolved, `isSelection` will work incorrectly for Elm. In particular, when selecting the entire file and beautifying it people would expect to see an automatically added module definition line, but this would not happen. Introducing `textIsNotEntireFile` solves this potential issue in Elm and in other languages.